packages/replayio/src/commands/record.ts

```diff
@@ -30,27 +31,26 @@ async function record(url: string = "about:blank") {
   if (recordingsNew.length > 0) {
     let selectedRecordings: LocalRecording[] = [];
     if (recordingsNew.length === 1) {
-      const confirmed = await confirm("Would you like to upload the new recording?", true);
+      const confirmed = await confirm(
+        "New recording found. Would you like to upload it?",
+        true,
+        "\n" + printRecordings(recordingsNew)
```

Comment:
This one feels a little bit too much to me. As the user, I focus on the last line of the printed content and that's now a row of the single-row table. YMMV and maybe that's just my initial reaction.

At the very least columns like Date and Status feel redundant here (it's always~ Now and it's always Recorded). Btw. this is true for the other branch too. In this particular branch the Process is likely redundant too (it's always going to be `(root)`, right? 🤔 ). I could be persuaded that this one is OK here for consistency's sake.

I find it weird that we print headers here (in the single-recording branch) but we don't do that in the multi-recording branch.

Reply:
> I could be persuaded that this one is OK here for consistency's sake.

That's what I was thinking. Consistency/familiarity (and a chance to verify that the host and duration look right).

> I find it weird that we print headers here (in the single-recording branch) but we don't do that in the multi-recording branch.

Fair. I can change that. Wasn't intentional.